docs/governance/index.md

```diff
@@ -4,28 +4,89 @@ summary: Hier staat een korte samenvatting over dit onderwerp
+Governance gaat over de inrichting van een organisatie en de daarbij horende processen, regels, gebruiken en bijbehorende verantwoordelijkheden. 
+Dit onderdeel van het Algoritmekader gaat over de governance van AI en algoritmes en systemen waarin deze verwerkt zijn. 
 
-Het Algoritmekader laat zien aan welke vereisten moet worden gedaan, hoe daar invulling aan kan worden gegeven (maatregelen) en welke rollen daar logischerwijs bij betrokken zijn. 
-Governance raakt in zekere zin al deze elementen doordat het deze, op verschillende niveaus, met elkaar verbindt.  
+!!! note "Let op!"
 
-In dit deel van het Algoritmekader worden de volgende zaken uitgewerkt; 
+    Governance wordt in verschillende contexten anders gebruikt, daarom spreken van “algoritme en AI governance”. 
+    Omdat het Algoritmekdader uitgaat van de [brede definitie van een algoritme](../overhetalgoritmekader/definities.md), hanteren we voor het gemak de term “algoritme-governance” als we het hebben over governance over algoritmes en AI. 
+    Dat kan omdat AI-systemen altijd algoritmes zijn, terwijl niet alle algoritmes AI-systemen zijn.
 
-- een verzameling praktijkvoorbeelden die tonen hoe organisaties hun governance kunnen inrichten om verantwoord met algoritmes en AI-systemen om te gaan. Daarbij worden goede praktijkvoorbeelden samengevat op hoofdlijnen; 
-- een handreiking hoe om te gaan met een governancestructuur die rekening houdt met verschillende niveaus van complexiteit van een organisatie.  
-- uitwerken van governance-maatregelen en koppeling aan rollen die hier logischerwijs bij passen, waarmee invulling kan worden gegeven aan een vereiste.  
+De noodzaak van governance is om grip te hebben (of krijgen) op algoritmes en AI binnen een organisatie en zaken als (onbedoelde) discriminatie te voorkomen. 
+Mede gezien de AI-verordening, die 1 augustus 2024 gefaseerd in werking is getreden, is het van belang dat organisaties zich organiseren om deze wet correct te kunnen uitvoeren. 
+Bewustzijn bij bestuur en organisatie is daarvoor een belangrijk vertrekpunt.
 
-De eerste twee punten zijn breed en geven algemene handvatten voor governance, het laatste punt gaat specifiek in op de vereisten/maatregelen. 
+### Verschillende niveau's van governance
+Algoritme en AI governance bestaat op verschillende niveaus: van (inter)nationaal niveau, naar organisatieniveau naar het niveau van het AI-systeem (zie Figuur hieronder). 
+In het Algoritmekader wordt gefocust op de twee niveaus: 
 
-Let wel: Het Algoritmekader schrijft niet voor hoe een organisatie ingericht moet worden qua eindverantwoordelijken/governance. We proberen met behulp van voorbeelden praktische handvatten te bieden over hoe organisaties dit kunnen inrichten.  
+- organisatie
+- algoritme (lifecycle).
+
+![Algoritme en AI governance ladder, TNO, 2024](https://github.com/user-attachments/assets/b7a237be-6caf-4266-910b-5fd05f4dc93b)
```

Comment:
Dit plaatje is wat mij betreft wat conflicterend met wat er gezegd wordt in de tekst. Volgens mij heeft het hele Algoritmekader de scope van zowel organisatie als systeem. Met de voorgestelde aanpassing van de levenscyclus wordt dat hopelijk ook duidelijker.

Reply:
Ik zie zelf wel een verschil tussen de kant van het AK die je operationeel gebruikt, en de kant van het AK die je gebruikt om je AI governance in te richten. Misschien niet passend bij dit plaatje, maar wel met die levenscyclus v2 inderdaad. 

De scope-aanduiding kan misschien beter weg. Ik zal de auteur van plaatje vragen of dat kan (die heeft hoop ik de ppt-versie)